src/content/tools/pub/pubspec.md

```diff
@@ -453,6 +453,11 @@ Pub.dev requires topics to follow these specifications:
 When choosing topics, consider if [existing topics]({{site.pub}}/topics)
 are relevant. Tagging with existing topics helps users discover your package.
 
+:::note
+On pub.dev topics may be merged into canonical topics. This aims to merge topics where the only difference is the spelling.
+You can contribute to the [list of canonical topics](https://github.com/dart-lang/pub-dev/blob/master/doc/topics.yaml) by filing a Pull-Request on Github.
+:::
```

Comment:
How about something like this? To  provide a little more detail about why they are merged.

```suggestion
:::note
Pub.dev might merge similar topics with different spellings into
a canonical topic to avoid duplication and improve searching by topic.

To contribute to the list of canonical topics and their aliases,
you can open a pull request editing the [`topics.yaml` file][] on GitHub.
:::

[`topics.yaml` file]: https://github.com/dart-lang/pub-dev/blob/master/doc/topics.yaml
```

Reply:
LGTM, did a few tiny tweaks,